dsgrid/cli/registry.py

```diff
@@ -251,18 +286,23 @@ def update_dimension(
 """
 
 
-@click.command(name="list")
+_list_dimension_mappings_epilog = """
+Examples:\n
+$ dsgrid registry dimension-mappings list\n
+$ dsgrid registry dimension-mappings list -f "Type [From, To] contains geography" -f "Submitter == username"\n
+"""
+
+
+@click.command(name="list", epilog=_list_dimension_mappings_epilog)
 @click.option(
     "-f",
     "--filter",
     multiple=True,
     type=str,
     help=f"""
-    filter table with a case-insensitive expression in the format 'field operation value',
-    accept multiple flags\b\n
+    Filter table with a case-insensitive expression in the format 'field operation value',
+    accepts multiple flags\b\n
```

Comment:
Similar to list command above, can we list the valid fields? (I would not have expected "Type [From, To]" to work ...